src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -27,6 +27,12 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; 'handler' functions for HTTP endpoints
 
+(defn handle-server-error
+  ([message]
+    (-> (rr/response message)
+        (rr/status 400)
+        (rr/content-type "text/plain"))))
+
```

Comment:
Ahh, yeah. I first created this as a fn that took zero arguments, then added a one arg form to use it in `handle-put-certificate-request!`. Then added a message to my usage of it in `web-routes`, deleted the zero arg form, but forgot to remove the surrounding parens from it. I did run the tests for this file at each step though... :\

I'll push an update with the extra parens removed.